DragaliaAPI/DragaliaAPI.Integration.Test/Dragalia/QuestReadStoryTest.cs

```diff
@@ -82,6 +82,38 @@ public async Task ReadStory_TheLonePaladyn_SetsTutorialStatus()
         response.UpdateDataList.UserData.TutorialStatus.Should().Be(10600);
     }
 
+    [Fact]
+    public async Task ReadStory_Chapter10Completion_GrantsRewards()
+    {
+        await this
+            .ApiContext.PlayerUserData.Where(x => x.ViewerId == this.ViewerId)
+            .ExecuteUpdateAsync(u => u.SetProperty(e => e.Level, 1).SetProperty(e => e.Exp, 0));
```

Comment:
I don't think this is realistic, as nobody would be level 1 while completing chapter 10. I would suggest setting level and XP to a pre-known arbitrary value and asserting that your exp afterwards is what you would expect after gaining 69990.